packages/wrangler/src/deployment-bundle/bundle-reporter.ts

```diff
@@ -6,7 +6,8 @@ import type { CfModule } from "./worker";
 import type { Metafile } from "esbuild";
 
 const ONE_KIB_BYTES = 1024;
-const ALLOWED_INITIAL_MAX = ONE_KIB_BYTES * 1024; // Current max is 1 MiB
+// Current max is 3 MiB for free accounts, 10 MiB for paid accounts.
```

Comment:
Could you add a link to https://developers.cloudflare.com/workers/platform/limits/#worker-size?

Reply:
added!